lavamoat/browserify/beta/policy.json

```diff
@@ -931,7 +931,6 @@
         "@metamask/snaps-utils": true,
         "@metamask/assets-controllers>@metamask/utils": true,
         "@metamask/name-controller>async-mutex": true,
-        "bitcoin-address-validation": true,
```

Comment:
I do not see where this has been deleted? 🤔

Reply:
Seems to have been deleted here: https://github.com/MetaMask/core/commit/e607b7fdaebce6333e02bc1946dec67b30ed7d29

Can probably be removed as a dependency on the assets-controllers package!"